tests/pytests/test_info_modules.py

```diff
@@ -64,7 +67,7 @@ def testInfoModulesBasic(env):
   env.assertEqual(field_info_to_dict(fieldsInfo['search_fields_tag']), get_search_field_info('Tag', 2, Sortable=1, CaseSensitive=1))
   env.assertEqual(field_info_to_dict(fieldsInfo['search_fields_numeric']), get_search_field_info('Numeric', 2, NoIndex=1))
   env.assertEqual(field_info_to_dict(fieldsInfo['search_fields_geo']), get_search_field_info('Geo', 1))
-  env.assertEqual(field_info_to_dict(fieldsInfo['search_fields_vector']), get_search_field_info('Vector', 2, Flat=1, HNSW=1))
+  env.assertEqual(field_info_to_dict(fieldsInfo['search_fields_vector']), get_search_field_info('Vector', 4, Flat=1, HNSW=1, SVS_VAMANA=2, SVS_VAMANA_Compressed=1))
```

Comment:
Do we have a test that also checks we count gc_marked_deleted_vectors and used_memory_vector_index correctly?
I suggest to incrementally add svs changes and making sure they take effect (for example, create an svs  index and ensure the `used_memory_vector_index` increased)

Reply:
Actually, I don't think that we count the marked deleted vector of svs as part of the vector stats today. We should address that and confirm it here after we implement it in vecsim. Regarding the used memory of vector indexes - it should be accounted - I will add a test

Reply:
Yep, svs should implement Tombstone
```
template <typename DataType, typename DistType>
VecSimIndexStatsInfo VecSimTieredIndex<DataType, DistType>::statisticInfo() const {
    auto stats = VecSimIndexStatsInfo{
        .memory = this->getAllocationSize(),
        .numberOfMarkedDeleted = 0, // Default value if cast fails
    };

    // If backend implements VecSimIndexTombstone, get number of marked deleted
    if (auto tombstone = dynamic_cast<VecSimIndexTombstone *>(this->backendIndex)) {
        stats.numberOfMarkedDeleted = tombstone->getNumMarkedDeleted();
    }

    return stats;
}
```